src/Spryker/Zed/SynchronizationBehavior/SynchronizationBehaviorConfig.php

```diff
@@ -43,7 +43,7 @@ public function isSynchronizationEnabled(): bool
      */
     public function isAliasKeysEnabled(): bool
     {
-        return false;
+        return true;
```

Comment:
Mb we can change this value on the project level only?
Not sure if we consider changing default config value in the core as a BC break or not

Reply:
How can this break something?

Reply:
It will create a new database column. We said that this is always a major, but as I already said, IMO having an optional column added is not a major.

I tested all stages and QA will also test all stages to be sure that this new column does not break projects.

@gechetspr Projects can do that already but it's a bug that it's not enabled by default. `sync:data` is not able to create the mapping data when set to false. To have full data you currently need to trigger all events to get the data properly synced into the storage.